### PR TITLE
improve video download tree usability

### PR DESCRIPTION
**Pull request by 66eli77**

### CODE REVIEW

**Comment by 66eli77** [comment]
@aronasorman  since you are the video downloading tester, I'm assigning this to you, but feel free to pass the torch

**Comment by aronasorman** [comment]
Will review by this week.

**Comment by aronasorman** [comment]
Any screenshots?

**Comment by 66eli77** [comment]
screen shot 2015-09-03 at 1 14 45 pm

**Comment by aronasorman** [comment]
I think @jtamiace is the better person to review this :)

**Comment by jtamiace** [comment]
Using more color to indicate status is a big improvement (thumbsup)

Is there another way to indicate whether or not an item has children though? 
The small arrow distinguishes it enough, but with the addition of the gray bar it gets hard to distinguish when there are a lot of folders open.
Using another distinguisher to separate folder nodes from content nodes would improve usability although it's pretty good as it is. Example: maybe folders could have bigger text than content nodes, or different colors, etc.

**Comment by 66eli77** [comment]
screen shot 2015-09-09 at 2 08 38 pm
@jtamiace  how about this

**Comment by 66eli77** [comment]
@jtamiace the fail is due to others change, feel free to merge if you felt good about the look

**Comment by rtibbles** [comment]
Partially downloaded folders are misaligned

image

**Comment by rtibbles** [comment]
As are complete ones:
image

File: kalite/updates/static/css/updates/update_videos.less [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import "../../../../../node_modules/jquery.fancytree/dist/skin-common.less";
 
 @fancy-inline-sprites: true;
-@fancy-image-dir: "../../../../../node_modules/jquery.fancytree/dist/skin-vista";
+@fancy-image-dir: "../../../../../node_modules/jquery.fancytree/dist/skin-Lion";
```
Comment (rtibbles) [inline kalite/updates/static/css/updates/update_videos.less]: Typo here - ```Lion``` should be ```lion```

**Comment by rtibbles** [comment]
A few issues to address before merge.

File: kalite/updates/static/css/updates/update_videos.less [hunk 2]
```diff
@@ -22,6 +22,44 @@
...
+    border-color: #d0d0d0;
+    border-style: solid;
+    border-width: 5px;
+}
+.fancytree-has-children {
+    border-color: #d0d0d0;
+    background-color: #d0d0d0;
+}
+.fancytree-has-children::before {
+    content: " \1F4C2";
```
Comment (rtibbles) [inline kalite/updates/static/css/updates/update_videos.less]: This doesn't render on my machine - switch out for glyphicons.

**Comment by 66eli77** [comment]
hide the fancytree-expander, using glyphicons instead
screen shot 2015-09-18 at 10 50 46 pm

**Comment by rtibbles** [comment]
Looks good, @66eli77. Merging.

**Pull request merged**